cartographer/mapping/global_trajectory_builder_interface.h

```diff
@@ -54,6 +55,8 @@ class GlobalTrajectoryBuilderInterface {
   virtual void AddRangefinderData(common::Time time,
                                   const Eigen::Vector3f& origin,
                                   const sensor::PointCloud& ranges) = 0;
+  virtual void AddLandmarkData(common::Time time,
```

Comment:
It's better to follow the pattern using `AddSensorData` which allows you to use the `Dispatchable<>` template.